diff --git a/bin/add-user.php b/bin/add-user.php
index 215d35d..afe4cc6 100644
--- a/bin/add-user.php
+++ b/bin/add-user.php
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Admin/autoload.php';
+$baseDir = '/usr/share/vpn-admin-portal';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\Config;
diff --git a/web/index.php b/web/index.php
index 30f1a16..a57c713 100644
--- a/web/index.php
+++ b/web/index.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Admin/autoload.php';
+$baseDir = '/usr/share/vpn-admin-portal';
 
 use fkooman\SeCookie\Cookie;
 use fkooman\SeCookie\Session;
