diff --git a/bin/certificate-info.php b/bin/certificate-info.php
index 61f6dab..74c3323 100644
--- a/bin/certificate-info.php
+++ b/bin/certificate-info.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use LC\Common\FileIO;
 
diff --git a/bin/generate-dns-zones.php b/bin/generate-dns-zones.php
index 01b9cec..dc0fe94 100644
--- a/bin/generate-dns-zones.php
+++ b/bin/generate-dns-zones.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use LC\Common\Config;
 use LC\Common\HttpClient\CurlHttpClient;
diff --git a/bin/generate-firewall.php b/bin/generate-firewall.php
index 9785cfc..20be226 100644
--- a/bin/generate-firewall.php
+++ b/bin/generate-firewall.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use LC\Common\Config;
 use LC\Common\FileIO;
diff --git a/bin/server-config.php b/bin/server-config.php
index 1a59896..bf311e0 100644
--- a/bin/server-config.php
+++ b/bin/server-config.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use LC\Common\Config;
 use LC\Common\HttpClient\CurlHttpClient;
diff --git a/libexec/client-connect.php b/libexec/client-connect.php
index b4cffde..c3501a2 100644
--- a/libexec/client-connect.php
+++ b/libexec/client-connect.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use LC\Common\Config;
 use LC\Common\HttpClient\CurlHttpClient;
diff --git a/libexec/client-disconnect.php b/libexec/client-disconnect.php
index a7c75b3..d61874e 100644
--- a/libexec/client-disconnect.php
+++ b/libexec/client-disconnect.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use LC\Common\Config;
 use LC\Common\HttpClient\CurlHttpClient;
