diff --git a/bin/add-user.php b/bin/add-user.php
index baa23c8..0084da0 100644
--- a/bin/add-user.php
+++ b/bin/add-user.php
@@ -8,9 +8,9 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-admin-portal';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Admin/autoload.php';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\Config;
diff --git a/web/index.php b/web/index.php
index 1ba95ae..ee2f3ab 100644
--- a/web/index.php
+++ b/web/index.php
@@ -7,9 +7,9 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-admin-portal';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Admin/autoload.php';
 
 use fkooman\SeCookie\Cookie;
 use fkooman\SeCookie\Session;
