diff --git a/bin/add-user.php b/bin/add-user.php
index baa23c8..e48489f 100644
--- a/bin/add-user.php
+++ b/bin/add-user.php
@@ -8,9 +8,9 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-user-portal';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\Config;
diff --git a/bin/foreign-key-list-fetcher.php b/bin/foreign-key-list-fetcher.php
index 30f210c..415a6d7 100644
--- a/bin/foreign-key-list-fetcher.php
+++ b/bin/foreign-key-list-fetcher.php
@@ -8,9 +8,9 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-user-portal';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
 
 use fkooman\OAuth\Client\Http\CurlHttpClient;
 use SURFnet\VPN\Common\CliParser;
diff --git a/bin/init.php b/bin/init.php
index 43827fe..e6a00d4 100644
--- a/bin/init.php
+++ b/bin/init.php
@@ -8,9 +8,9 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-user-portal';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\FileIO;
diff --git a/bin/show-public-key.php b/bin/show-public-key.php
index 433fb68..73567dc 100644
--- a/bin/show-public-key.php
+++ b/bin/show-public-key.php
@@ -8,9 +8,9 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-user-portal';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\FileIO;
diff --git a/web/api.php b/web/api.php
index ffd32bd..79e2679 100644
--- a/web/api.php
+++ b/web/api.php
@@ -7,9 +7,9 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-user-portal';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
 
 use fkooman\OAuth\Server\BearerValidator;
 use fkooman\OAuth\Server\ClientInfo;
diff --git a/web/index.php b/web/index.php
index 85549ff..cf3eae0 100644
--- a/web/index.php
+++ b/web/index.php
@@ -7,9 +7,9 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-user-portal';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
 
 use fkooman\OAuth\Client\Http\CurlHttpClient as OAuthCurlHttpClient;
 use fkooman\OAuth\Client\OAuthClient;
diff --git a/web/oauth.php b/web/oauth.php
index 7625396..a50ca8d 100644
--- a/web/oauth.php
+++ b/web/oauth.php
@@ -7,9 +7,9 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-user-portal';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
 
 use fkooman\OAuth\Server\ClientInfo;
 use fkooman\OAuth\Server\OAuthServer;
