diff --git a/bin/disconnect-expired-certificates.php b/bin/disconnect-expired-certificates.php
index 35d0c80..39b906e 100644
--- a/bin/disconnect-expired-certificates.php
+++ b/bin/disconnect-expired-certificates.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Server/autoload.php';
+$baseDir = '/usr/share/vpn-server-api';
 
 use LC\Common\Config;
 use LC\Common\Logger;
diff --git a/bin/housekeeping.php b/bin/housekeeping.php
index 93a5a19..fefe00b 100644
--- a/bin/housekeeping.php
+++ b/bin/housekeeping.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Server/autoload.php';
+$baseDir = '/usr/share/vpn-server-api';
 
 use LC\Server\Storage;
 
diff --git a/bin/init.php b/bin/init.php
index 53746a8..64c429e 100644
--- a/bin/init.php
+++ b/bin/init.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Server/autoload.php';
+$baseDir = '/usr/share/vpn-server-api';
 
 use LC\Server\CA\EasyRsaCa;
 use LC\Server\Storage;
diff --git a/bin/stats.php b/bin/stats.php
index c898aa8..d7d22dd 100644
--- a/bin/stats.php
+++ b/bin/stats.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Server/autoload.php';
+$baseDir = '/usr/share/vpn-server-api';
 
 use LC\Common\Config;
 use LC\Common\FileIO;
diff --git a/bin/status.php b/bin/status.php
index 330f891..f408a1e 100644
--- a/bin/status.php
+++ b/bin/status.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Server/autoload.php';
+$baseDir = '/usr/share/vpn-server-api';
 
 use LC\Common\Config;
 use LC\Common\Logger;
diff --git a/bin/update-api-secrets.php b/bin/update-api-secrets.php
index e28053d..d8f82ba 100644
--- a/bin/update-api-secrets.php
+++ b/bin/update-api-secrets.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Server/autoload.php';
+$baseDir = '/usr/share/vpn-server-api';
 
 use LC\Common\CliParser;
 use LC\Common\Config;
diff --git a/bin/update-ip.php b/bin/update-ip.php
index aace998..9b0875f 100644
--- a/bin/update-ip.php
+++ b/bin/update-ip.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Server/autoload.php';
+$baseDir = '/usr/share/vpn-server-api';
 
 use LC\Common\CliParser;
 use LC\Common\Config;
diff --git a/web/api.php b/web/api.php
index b05e796..bc5228e 100644
--- a/web/api.php
+++ b/web/api.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Server/autoload.php';
+$baseDir = '/usr/share/vpn-server-api';
 
 use LC\Common\Config;
 use LC\Common\Http\BasicAuthenticationHook;
