diff --git a/bin/housekeeping.php b/bin/housekeeping.php
index fc7bf05..73a3553 100644
--- a/bin/housekeeping.php
+++ b/bin/housekeeping.php
@@ -7,9 +7,9 @@
  * Copyright: 2016-2017, The Commons Conservancy eduVPN Programme
  * SPDX-License-Identifier: AGPL-3.0+
  */
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-server-api';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Server/autoload.php';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Server\Storage;
diff --git a/bin/init.php b/bin/init.php
index 8e73d13..b1ac2b6 100644
--- a/bin/init.php
+++ b/bin/init.php
@@ -7,9 +7,9 @@
  * Copyright: 2016-2017, The Commons Conservancy eduVPN Programme
  * SPDX-License-Identifier: AGPL-3.0+
  */
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-server-api';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Server/autoload.php';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\Config;
diff --git a/bin/show-instance-info.php b/bin/show-instance-info.php
index 5999f01..1af1a39 100644
--- a/bin/show-instance-info.php
+++ b/bin/show-instance-info.php
@@ -7,9 +7,9 @@
  * Copyright: 2016-2017, The Commons Conservancy eduVPN Programme
  * SPDX-License-Identifier: AGPL-3.0+
  */
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-server-api';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Server/autoload.php';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\Config;
diff --git a/bin/stats.php b/bin/stats.php
index a0fb2f8..4580a88 100644
--- a/bin/stats.php
+++ b/bin/stats.php
@@ -7,9 +7,9 @@
  * Copyright: 2016-2017, The Commons Conservancy eduVPN Programme
  * SPDX-License-Identifier: AGPL-3.0+
  */
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-server-api';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Server/autoload.php';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\FileIO;
diff --git a/bin/status.php b/bin/status.php
index 8266bcb..7ea13f4 100644
--- a/bin/status.php
+++ b/bin/status.php
@@ -7,9 +7,9 @@
  * Copyright: 2016-2017, The Commons Conservancy eduVPN Programme
  * SPDX-License-Identifier: AGPL-3.0+
  */
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-server-api';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Server/autoload.php';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\Config;
diff --git a/bin/update-api-secrets.php b/bin/update-api-secrets.php
index c8aee96..207d207 100644
--- a/bin/update-api-secrets.php
+++ b/bin/update-api-secrets.php
@@ -7,9 +7,9 @@
  * Copyright: 2016-2017, The Commons Conservancy eduVPN Programme
  * SPDX-License-Identifier: AGPL-3.0+
  */
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-server-api';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Server/autoload.php';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\Config;
diff --git a/bin/update-ip.php b/bin/update-ip.php
index 6330a94..9e2531f 100644
--- a/bin/update-ip.php
+++ b/bin/update-ip.php
@@ -7,9 +7,9 @@
  * Copyright: 2016-2017, The Commons Conservancy eduVPN Programme
  * SPDX-License-Identifier: AGPL-3.0+
  */
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-server-api';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Server/autoload.php';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\Config;
diff --git a/web/api.php b/web/api.php
index 41aa124..911107f 100644
--- a/web/api.php
+++ b/web/api.php
@@ -6,9 +6,9 @@
  * Copyright: 2016-2017, The Commons Conservancy eduVPN Programme
  * SPDX-License-Identifier: AGPL-3.0+
  */
-$baseDir = dirname(__DIR__);
+$baseDir = '/usr/share/vpn-server-api';
 /** @psalm-suppress UnresolvableInclude */
-require_once sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/SURFnet/VPN/Server/autoload.php';
 
 use fkooman\OAuth\Client\Http\CurlHttpClient;
 use fkooman\OAuth\Client\OAuthClient;
