diff --git a/web/index.php b/web/index.php
index 188434c..9f00081 100644
--- a/web/index.php
+++ b/web/index.php
@@ -22,9 +22,9 @@
  * SOFTWARE.
  */
 
-$baseDir = \dirname(__DIR__);
+$baseDir = '/usr/share/php-voot2-provider';
 /** @psalm-suppress UnresolvableInclude */
-require_once \sprintf('%s/vendor/autoload.php', $baseDir);
+require_once '/usr/share/php/fkooman/Voot/autoload.php';
 
 use fkooman\Voot\Config;
 use fkooman\Voot\ErrorLogger;
