diff --git a/bin/generate.php b/bin/generate.php
index 9d4079a..80b88ba 100644
--- a/bin/generate.php
+++ b/bin/generate.php
@@ -22,8 +22,8 @@
  * SOFTWARE.
  */
 
-require_once \dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = \dirname(__DIR__);
+require_once '/usr/share/php/fkooman/SAML/DS/autoload.php';
+$baseDir = '/usr/share/php-saml-ds';
 
 use fkooman\SAML\DS\Config;
 use fkooman\SAML\DS\HttpClient\CurlHttpClient;
diff --git a/web/index.php b/web/index.php
index 793d9aa..d5c15dd 100644
--- a/web/index.php
+++ b/web/index.php
@@ -22,8 +22,8 @@
  * SOFTWARE.
  */
 
-require_once \dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = \dirname(__DIR__);
+require_once '/usr/share/php/fkooman/SAML/DS/autoload.php';
+$baseDir = '/usr/share/php-saml-ds';
 
 use fkooman\SAML\DS\Config;
 use fkooman\SAML\DS\Http\Request;
