diff --git a/web/metadata.php b/web/metadata.php
index f3b422b..b0b5c24 100644
--- a/web/metadata.php
+++ b/web/metadata.php
@@ -22,13 +22,13 @@
  * SOFTWARE.
  */
 
-require_once \dirname(__DIR__).'/vendor/autoload.php';
+require_once '/usr/share/php/fkooman/SAML/IdP/autoload.php';
 
 use fkooman\SAML\IdP\Certificate;
 use fkooman\SAML\IdP\Http\Request;
 use fkooman\SAML\IdP\Http\Response;
 
-$baseDir = \dirname(__DIR__);
+$baseDir = '/usr/share/php-saml-idp';
 $rsaCert = Certificate::fromFile($baseDir.'/config/server.crt');
 $keyInfo = $rsaCert->toKeyInfo();
 
diff --git a/web/sso.php b/web/sso.php
index 56372ef..00f6cb7 100644
--- a/web/sso.php
+++ b/web/sso.php
@@ -22,7 +22,7 @@
  * SOFTWARE.
  */
 
-require_once \dirname(__DIR__).'/vendor/autoload.php';
+require_once '/usr/share/php/fkooman/SAML/IdP/autoload.php';
 
 use fkooman\SAML\IdP\Certificate;
 use fkooman\SAML\IdP\Config;
@@ -33,7 +33,7 @@ use ParagonIE\ConstantTime\Base64;
 
 \libxml_disable_entity_loader(true);
 
-$baseDir = \dirname(__DIR__);
+$baseDir = '/usr/share/php-saml-idp';
 
 try {
     $config = Config::fromFile($baseDir.'/config/config.php');
