diff --git a/bin/add-user.php b/bin/add-user.php
index 7d36dff..2f9ba78 100644
--- a/bin/add-user.php
+++ b/bin/add-user.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LetsConnect\Common\CliParser;
 use LetsConnect\Common\Config;
diff --git a/bin/foreign-key-list-fetcher.php b/bin/foreign-key-list-fetcher.php
index 0160081..8359a2f 100644
--- a/bin/foreign-key-list-fetcher.php
+++ b/bin/foreign-key-list-fetcher.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LetsConnect\Common\Config;
 use LetsConnect\Portal\ForeignKeyListFetcher;
diff --git a/bin/init.php b/bin/init.php
index f12077b..613b25a 100644
--- a/bin/init.php
+++ b/bin/init.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LetsConnect\Common\FileIO;
 use LetsConnect\Portal\Storage;
diff --git a/bin/show-public-key.php b/bin/show-public-key.php
index bb06348..020c9e5 100644
--- a/bin/show-public-key.php
+++ b/bin/show-public-key.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LetsConnect\Common\FileIO;
 
diff --git a/web/api.php b/web/api.php
index 0ea57fd..cb31ae9 100644
--- a/web/api.php
+++ b/web/api.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\BearerValidator;
 use fkooman\OAuth\Server\LocalSigner;
diff --git a/web/index.php b/web/index.php
index 950c2b0..a13a33c 100644
--- a/web/index.php
+++ b/web/index.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\LocalSigner;
 use fkooman\OAuth\Server\OAuthServer;
diff --git a/web/oauth.php b/web/oauth.php
index af31a15..0abfffa 100644
--- a/web/oauth.php
+++ b/web/oauth.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\LocalSigner;
 use fkooman\OAuth\Server\OAuthServer;
