diff --git a/bin/certificate-info.php b/bin/certificate-info.php
index 0013e4f..40c9ecb 100644
--- a/bin/certificate-info.php
+++ b/bin/certificate-info.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use LetsConnect\Common\FileIO;
 
diff --git a/bin/generate-firewall.php b/bin/generate-firewall.php
index a1f9204..18d87cd 100644
--- a/bin/generate-firewall.php
+++ b/bin/generate-firewall.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use LetsConnect\Common\CliParser;
 use LetsConnect\Common\Config;
diff --git a/bin/server-config.php b/bin/server-config.php
index bce6893..6da66d5 100644
--- a/bin/server-config.php
+++ b/bin/server-config.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use LetsConnect\Common\Config;
 use LetsConnect\Common\HttpClient\CurlHttpClient;
diff --git a/libexec/client-connect.php b/libexec/client-connect.php
index a6df3d5..5b20411 100644
--- a/libexec/client-connect.php
+++ b/libexec/client-connect.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use LetsConnect\Common\Config;
 use LetsConnect\Common\HttpClient\CurlHttpClient;
diff --git a/libexec/client-disconnect.php b/libexec/client-disconnect.php
index ad65347..c1ecaf1 100644
--- a/libexec/client-disconnect.php
+++ b/libexec/client-disconnect.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use LetsConnect\Common\Config;
 use LetsConnect\Common\HttpClient\CurlHttpClient;
