diff --git a/bin/housekeeping.php b/bin/housekeeping.php
index e7ff7cd..1990086 100644
--- a/bin/housekeeping.php
+++ b/bin/housekeeping.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Server/autoload.php';
+$baseDir = '/usr/share/vpn-server-api';
 
 use LetsConnect\Server\Storage;
 
diff --git a/bin/init.php b/bin/init.php
index aaf18b7..fbddea8 100644
--- a/bin/init.php
+++ b/bin/init.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Server/autoload.php';
+$baseDir = '/usr/share/vpn-server-api';
 
 use LetsConnect\Common\Config;
 use LetsConnect\Server\CA\EasyRsaCa;
diff --git a/bin/stats.php b/bin/stats.php
index b8996cb..14e599f 100644
--- a/bin/stats.php
+++ b/bin/stats.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Server/autoload.php';
+$baseDir = '/usr/share/vpn-server-api';
 
 use LetsConnect\Common\Config;
 use LetsConnect\Common\FileIO;
diff --git a/bin/status.php b/bin/status.php
index 4a73982..8bef619 100644
--- a/bin/status.php
+++ b/bin/status.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Server/autoload.php';
+$baseDir = '/usr/share/vpn-server-api';
 
 use LC\OpenVpn\ManagementSocket;
 use LetsConnect\Common\Config;
diff --git a/bin/update-api-secrets.php b/bin/update-api-secrets.php
index 68e7684..5dfde18 100644
--- a/bin/update-api-secrets.php
+++ b/bin/update-api-secrets.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Server/autoload.php';
+$baseDir = '/usr/share/vpn-server-api';
 
 use LetsConnect\Common\CliParser;
 use LetsConnect\Common\Config;
diff --git a/bin/update-ip.php b/bin/update-ip.php
index 4089bfc..36432a8 100644
--- a/bin/update-ip.php
+++ b/bin/update-ip.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Server/autoload.php';
+$baseDir = '/usr/share/vpn-server-api';
 
 use LetsConnect\Common\CliParser;
 use LetsConnect\Common\Config;
diff --git a/web/api.php b/web/api.php
index 50e2643..08cc30f 100644
--- a/web/api.php
+++ b/web/api.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Server/autoload.php';
+$baseDir = '/usr/share/vpn-server-api';
 
 use LC\OpenVpn\ManagementSocket;
 use LetsConnect\Common\Config;
