diff --git a/bin/add-user.php b/bin/add-user.php
index b80af68..6cf9d3f 100644
--- a/bin/add-user.php
+++ b/bin/add-user.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LetsConnect\Common\CliParser;
 use LetsConnect\Common\Config;
diff --git a/bin/foreign-key-list-fetcher.php b/bin/foreign-key-list-fetcher.php
index 0160081..87c8742 100644
--- a/bin/foreign-key-list-fetcher.php
+++ b/bin/foreign-key-list-fetcher.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LetsConnect\Common\Config;
 use LetsConnect\Portal\ForeignKeyListFetcher;
diff --git a/bin/init.php b/bin/init.php
index ccc5381..dd96363 100644
--- a/bin/init.php
+++ b/bin/init.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LetsConnect\Common\FileIO;
 use LetsConnect\Common\HttpClient\CurlHttpClient;
diff --git a/bin/show-public-key.php b/bin/show-public-key.php
index bb06348..904df63 100644
--- a/bin/show-public-key.php
+++ b/bin/show-public-key.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LetsConnect\Common\FileIO;
 
diff --git a/web/api.php b/web/api.php
index a9ace84..814a46b 100644
--- a/web/api.php
+++ b/web/api.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\BearerValidator;
 use fkooman\OAuth\Server\SodiumSigner;
diff --git a/web/index.php b/web/index.php
index e2ec585..36799c4 100644
--- a/web/index.php
+++ b/web/index.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\OAuthServer;
 use fkooman\OAuth\Server\SodiumSigner;
diff --git a/web/oauth.php b/web/oauth.php
index 44900f8..69fe370 100644
--- a/web/oauth.php
+++ b/web/oauth.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\OAuthServer;
 use fkooman\OAuth\Server\SodiumSigner;
