diff --git a/bin/add-user.php b/bin/add-user.php
index e7b7915..10b77e7 100644
--- a/bin/add-user.php
+++ b/bin/add-user.php
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\Config;
diff --git a/bin/foreign-key-list-fetcher.php b/bin/foreign-key-list-fetcher.php
index 8b78b36..08d4727 100644
--- a/bin/foreign-key-list-fetcher.php
+++ b/bin/foreign-key-list-fetcher.php
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use SURFnet\VPN\Common\Config;
 use SURFnet\VPN\Portal\ForeignKeyListFetcher;
diff --git a/bin/init.php b/bin/init.php
index a0a014f..852bda1 100644
--- a/bin/init.php
+++ b/bin/init.php
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use SURFnet\VPN\Common\FileIO;
 use SURFnet\VPN\Common\HttpClient\CurlHttpClient;
diff --git a/bin/show-public-key.php b/bin/show-public-key.php
index 5a5f414..426e02b 100644
--- a/bin/show-public-key.php
+++ b/bin/show-public-key.php
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use SURFnet\VPN\Common\FileIO;
 
diff --git a/web/api.php b/web/api.php
index 0cdcdfe..2dd9b9a 100644
--- a/web/api.php
+++ b/web/api.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\BearerValidator;
 use fkooman\OAuth\Server\SodiumSigner;
diff --git a/web/index.php b/web/index.php
index 21d547b..3ec7054 100644
--- a/web/index.php
+++ b/web/index.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\OAuthServer;
 use fkooman\OAuth\Server\SodiumSigner;
diff --git a/web/oauth.php b/web/oauth.php
index 0b68be4..97f2e4c 100644
--- a/web/oauth.php
+++ b/web/oauth.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\OAuthServer;
 use fkooman\OAuth\Server\SodiumSigner;
