diff --git a/bin/add-user.php b/bin/add-user.php
index 215d35d..0a58cb5 100644
--- a/bin/add-user.php
+++ b/bin/add-user.php
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\Config;
diff --git a/bin/foreign-key-list-fetcher.php b/bin/foreign-key-list-fetcher.php
index 73e2815..b3fd92e 100644
--- a/bin/foreign-key-list-fetcher.php
+++ b/bin/foreign-key-list-fetcher.php
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Client\Http\CurlHttpClient;
 use SURFnet\VPN\Common\CliParser;
diff --git a/bin/generate-voucher.php b/bin/generate-voucher.php
index a2fe968..dba230a 100644
--- a/bin/generate-voucher.php
+++ b/bin/generate-voucher.php
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\Http\PdoAuth;
diff --git a/bin/init.php b/bin/init.php
index aeb1bd1..670c12e 100644
--- a/bin/init.php
+++ b/bin/init.php
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\FileIO;
diff --git a/bin/show-public-key.php b/bin/show-public-key.php
index fea3f4a..335a396 100644
--- a/bin/show-public-key.php
+++ b/bin/show-public-key.php
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\FileIO;
diff --git a/web/api.php b/web/api.php
index face6f0..f7edc24 100644
--- a/web/api.php
+++ b/web/api.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\BearerValidator;
 use fkooman\OAuth\Server\SodiumSigner;
diff --git a/web/index.php b/web/index.php
index c7caab4..0836e71 100644
--- a/web/index.php
+++ b/web/index.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Client\Http\CurlHttpClient as OAuthCurlHttpClient;
 use fkooman\OAuth\Client\OAuthClient;
diff --git a/web/oauth.php b/web/oauth.php
index 37a370b..2042a54 100644
--- a/web/oauth.php
+++ b/web/oauth.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\OAuthServer;
 use fkooman\OAuth\Server\SodiumSigner;
