diff --git a/bin/certificate-info.php b/bin/certificate-info.php
index 3a295b8..98c468b 100644
--- a/bin/certificate-info.php
+++ b/bin/certificate-info.php
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use SURFnet\VPN\Common\FileIO;
 
diff --git a/bin/generate-firewall.php b/bin/generate-firewall.php
index cdc4928..6b19b14 100644
--- a/bin/generate-firewall.php
+++ b/bin/generate-firewall.php
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use SURFnet\VPN\Common\CliParser;
 use SURFnet\VPN\Common\Config;
diff --git a/bin/server-config.php b/bin/server-config.php
index 864287a..942977b 100644
--- a/bin/server-config.php
+++ b/bin/server-config.php
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use SURFnet\VPN\Common\Config;
 use SURFnet\VPN\Common\HttpClient\CurlHttpClient;
diff --git a/libexec/client-connect.php b/libexec/client-connect.php
index ae768f0..e4bc6cf 100644
--- a/libexec/client-connect.php
+++ b/libexec/client-connect.php
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use SURFnet\VPN\Common\Config;
 use SURFnet\VPN\Common\HttpClient\CurlHttpClient;
diff --git a/libexec/client-disconnect.php b/libexec/client-disconnect.php
index 30fa447..49de71c 100644
--- a/libexec/client-disconnect.php
+++ b/libexec/client-disconnect.php
@@ -8,8 +8,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/SURFnet/VPN/Node/autoload.php';
+$baseDir = '/usr/share/vpn-server-node';
 
 use SURFnet\VPN\Common\Config;
 use SURFnet\VPN\Common\HttpClient\CurlHttpClient;
