diff --git a/bin/add-user.php b/bin/add-user.php
index 7d36dff..2f9ba78 100644
--- a/bin/add-user.php
+++ b/bin/add-user.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LetsConnect\Common\CliParser;
 use LetsConnect\Common\Config;
diff --git a/bin/foreign-key-list-fetcher.php b/bin/foreign-key-list-fetcher.php
index 0160081..8359a2f 100644
--- a/bin/foreign-key-list-fetcher.php
+++ b/bin/foreign-key-list-fetcher.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LetsConnect\Common\Config;
 use LetsConnect\Portal\ForeignKeyListFetcher;
diff --git a/bin/generate-key.php b/bin/generate-key.php
index cbf129f..da16ba5 100644
--- a/bin/generate-key.php
+++ b/bin/generate-key.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LetsConnect\Common\FileIO;
 use LetsConnect\Portal\OAuth\Keys\SecretKey;
diff --git a/bin/init.php b/bin/init.php
index f251ef7..682a9ba 100644
--- a/bin/init.php
+++ b/bin/init.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LetsConnect\Common\FileIO;
 use LetsConnect\Portal\Storage;
diff --git a/web/api.php b/web/api.php
index f2e4a97..348633d 100644
--- a/web/api.php
+++ b/web/api.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\BearerValidator;
 use LetsConnect\Common\Config;
diff --git a/web/index.php b/web/index.php
index 6059d4b..fd36d89 100644
--- a/web/index.php
+++ b/web/index.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\OAuthServer;
 use fkooman\SAML\SP\PrivateKey;
diff --git a/web/oauth.php b/web/oauth.php
index db39ec4..ac04817 100644
--- a/web/oauth.php
+++ b/web/oauth.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LetsConnect/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\OAuthServer;
 use LetsConnect\Common\Config;
