diff --git a/bin/add-user.php b/bin/add-user.php
index cd79e62..05a40d7 100644
--- a/bin/add-user.php
+++ b/bin/add-user.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LC\Common\CliParser;
 use LC\Common\Config;
diff --git a/bin/foreign-key-list-fetcher.php b/bin/foreign-key-list-fetcher.php
index 05cd7b3..3c57e8a 100644
--- a/bin/foreign-key-list-fetcher.php
+++ b/bin/foreign-key-list-fetcher.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LC\Common\Config;
 use LC\Portal\ForeignKeyListFetcher;
diff --git a/bin/generate-oauth-key.php b/bin/generate-oauth-key.php
index da9e661..2445e25 100644
--- a/bin/generate-oauth-key.php
+++ b/bin/generate-oauth-key.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\Jwt\Keys\EdDSA\SecretKey;
 use LC\Common\FileIO;
diff --git a/bin/init.php b/bin/init.php
index 0e41697..b93d23b 100644
--- a/bin/init.php
+++ b/bin/init.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LC\Common\Config;
 use LC\Common\FileIO;
diff --git a/bin/show-oauth-key.php b/bin/show-oauth-key.php
index d528642..9a7a07b 100644
--- a/bin/show-oauth-key.php
+++ b/bin/show-oauth-key.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\Jwt\Keys\EdDSA\SecretKey;
 use LC\Common\FileIO;
diff --git a/web/api.php b/web/api.php
index 10c1f7f..1cfc1a6 100644
--- a/web/api.php
+++ b/web/api.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\Jwt\Keys\EdDSA\SecretKey;
 use LC\Common\Config;
diff --git a/web/index.php b/web/index.php
index c26a929..42272e3 100644
--- a/web/index.php
+++ b/web/index.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\Jwt\Keys\EdDSA\SecretKey;
 use fkooman\OAuth\Server\OAuthServer;
diff --git a/web/oauth.php b/web/oauth.php
index 6984276..e131fdd 100644
--- a/web/oauth.php
+++ b/web/oauth.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\Jwt\Keys\EdDSA\SecretKey;
 use fkooman\OAuth\Server\OAuthServer;
